pkg/operator/discover/discover.go

```diff
@@ -77,6 +79,14 @@ func (d *Discover) Start(namespace, discoverImage, securityAccount string) error
 
 func (d *Discover) createDiscoverDaemonSet(namespace, discoverImage, securityAccount string) error {
 	privileged := true
+	discovery_parameters := []string{"discover",
+		"--discover-interval", getEnvVar(discoverIntervalEnv, "60m")}
+	use_cv, err := strconv.ParseBool(getEnvVar(discoverUsesCV, "false"))
+	if err == nil {
+		if use_cv {
```

Comment:
to go with earlier comments, `use_cv` should be a parameter to this method. When the Ceph operator calls `discover.Start()`, it will pass `true`, while other providers will pass `false.`

Reply:
`use-ceph-volume` changed to parameter , not an `env-var` anymore